### PR TITLE
Update analysis_center page

### PR DESCRIPTION
**Pull request by clsackett**
Moving PR from gdcv2_doc_drafts to gdc-docs develop-v2 branch

- Copied information on the Analysis Center from the Quick Start page to the Analysis Center page per https://gdc-ctds.atlassian.net/browse/DOC-263
- Also added links to doc pages for the analysis tools except Gene Expression Clustering

### CODE REVIEW

File: docs/Data_Portal/Users_Guide/analysis_center.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+
+* A teal 'Play' button to launch the analysis tool on the given cohort
+* A 'Demo' button that launches a demonstration of the analysis tool on an example cohort
+* Clicking on the name of the analysis tool in the tool card toggles a drop down description of the analysis tool
+* The number of cases from the cohort that the analysis will be performed on is at the bottom of the card
 
 ## Core Tools ##
 
-This section contains the core GDC tools.  This includes the [Projects](LINKHERE) tool, the [Cohort Builder](LINKHERE), and the [Repository](LINKHERE). These can be selected for use with your current cohort by clicking on each from the Core Tools section.
+This section contains the core GDC tools.  This includes the [Projects](https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/docs/Data_Portal/Users_Guide/Projects.md) tool, the [Cohort Builder](https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/docs/Data_Portal/Users_Guide/cohort_builder.md), and the [Repository](https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/docs/Data_Portal/Users_Guide/Repository.md). These can be selected for use with your current cohort by clicking on each from the Core Tools section.
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/analysis_center.md:19]: I actually have no idea how these links would behave on our docs site and User Services / Bill would know better than I would. I looked at some past examples and it seems we usually just did something like `[Projects](Projects.md)`? 🤷‍♀️

Comment (clsackett) [reply docs/Data_Portal/Users_Guide/analysis_center.md]: Ah gotcha, thanks for pointing that out! I made a new commit with changes to the links

**Review by wteouchicago — APPROVED** [review]
looks good to me. deferring to @wwysoc2 on how the links should work since I've never done those myself.

**Review by wwysoc2 — APPROVED** [review]
Links and everything else look good.

**Pull request merged**